### PR TITLE
Introduce new attribute for RH Docs and fixes

### PR DESCRIPTION
**Pull request by mjivraja**
With this PR, we have introduced the new attributes for RHEL Docs links. In addition, it will help address the sub-domain name change in the future. The links are tested with new attributes are limited and will be covered further once this PR gets approval for its implementation.

Also, I have addressed the suggestions from Aneta from the PR https://github.com/theforeman/foreman-documentation/pull/2805. This is mostly about fixing links of RHEL with proper titles of guides and references.

Please cherry-pick my commits into:

* [X] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/attributes-satellite.adoc [hunk 1]
```diff
@@ -33,6 +33,7 @@
 :HammerDocURL: {BaseURL}hammer_cli_guide/index#
 :ConfiguringVMSubscriptionsDocURL: {BaseURL}configuring_virtual_machine_subscriptions/index#
 :ConversionsToolkitDocURL: {BaseURL}converting_hosts_to_rhel_by_using_satellite_conversions_toolkit/index#
+:RHSupportURL:
```
Comment (Lennonka) [inline guides/common/attributes-satellite.adoc:36]: Why is the value empty?

**Review by Lennonka — CHANGES_REQUESTED** [review]
Link replacements look good. However, I'm not sure about the other changes. See comments below.

File: guides/common/modules/proc_configuring-repositories-proxy.adoc [hunk 2]
```diff
@@ -36,7 +34,7 @@ Select the operating system and version you are installing on:
 --enable={RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
 ----
 
-include::snip_step-verify-enabled-repolist.adoc[]
+include::snip_verification-enabled-repolist.adoc[]
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/proc_configuring-repositories-proxy.adoc]: Addressed :+1:

File: guides/common/attributes-satellite.adoc (hunk 1 above)
Comment (mjivraja) [reply guides/common/attributes-satellite.adoc]: Redundant. Removed :+1:

File: guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc [hunk 3]
```diff
@@ -2,7 +2,7 @@
 = Kerberos configuration in web browsers
 
 ifndef::orcharhino[]
-For information on configuring Firefox, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in the _{RHEL} Configuring authentication and authorization in RHEL_ guide.
+For information about configuring Firefox, see {RHELDocsBaseURL}9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in _Configuring authentication and authorization in RHEL_ for {RHEL} 9.
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc:5]: Should I apply the above format? I was on PTO and I am not sure if this was discussed.

cc: @asteflova

[the remark this answers is not in the document]
Comment (asteflova) [reply guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc:5]: I don't think we discussed this. But I agree with Lena that it's best to keep whatever format is used for any other project guide, like Foreman guides themselves.

My 2 cents: No need for a separate discussion, this PR should be enough to come to an agreement.

[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc]: Addressed :+1:

File: guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc [hunk 4]
```diff
@@ -2,7 +2,7 @@
 = Kerberos configuration in web browsers
 
 ifndef::orcharhino[]
-For information on configuring Firefox, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in the _{RHEL} Configuring authentication and authorization in RHEL_ guide.
+For information about configuring Firefox, see {RHELDocsBaseURL}9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in the _{RHEL}{nbsp}9 Configuring authentication and authorization in RHEL_.
```
Comment (Lennonka) [inline guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc:5]:
```suggestion
For information about configuring Firefox, see {RHELDocsBaseURL}9/html/configuring_authentication_and_authorization_in_rhel/configuring_applications_for_sso#Configuring_Firefox_to_use_Kerberos_for_SSO[Configuring Firefox to use Kerberos for single sign-on] in _{RHEL}{nbsp}9 Configuring authentication and authorization in RHEL_.
```
Let's not use "the" when the name of the guide doesn't end in "guide".
Applies to all occurences in the whole PR.

File: guides/common/modules/con_using-freeipa.adoc [hunk 5]
```diff
@@ -19,6 +19,6 @@ include::snip_do-not-use-both-ldap-and-freeipa.adoc[]
 
 The examples in this chapter assume separation between {FreeIPA} and {Project} configuration.
 ifndef::orcharhino[]
-However, if you have administrator privileges for both servers, you can configure {FreeIPA} as described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/installing_identity_management/index[{RHEL} 9 Installing Identity Management Guide].
-For {RHEL} 8, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/installing_identity_management/index[Installing Identity Management Guide].
+However, if you have administrator privileges for both servers, you can configure {FreeIPA} as described in {RHELDocsBaseURL}9/html-single/installing_identity_management/index[Installing Identity Management] for _{RHEL}{nbsp}9_.
+See {RHELDocsBaseURL}8/html-single/installing_identity_management/index[Installing Identity Management Guide] for _{RHEL}{nbsp}8_.
```
Comment (Lennonka) [inline guides/common/modules/con_using-freeipa.adoc:23]:
```suggestion
However, if you have administrator privileges for both servers, you can configure {FreeIPA} as described in {RHELDocsBaseURL}9/html-single/installing_identity_management/index[_{RHEL}{nbsp}9 Installing Identity Management_] or {RHELDocsBaseURL}8/html-single/installing_identity_management/index[_{RHEL}{nbsp}8 Installing Identity Management Guide_].
```

File: guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc [hunk 6]
```diff
@@ -29,7 +29,7 @@ The generated one-time password must be used on the client to complete {FreeIPA}
 ====
 +
 ifdef::satellite[]
-For more information on host configuration properties, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_and_managing_identity_management/index#con_host-entry-LDAP_managing-hosts-ui[Host entry in IdM LDAP] in _Configuring and managing Identity Management_.
+For more information on host configuration properties, see {RHELDocsBaseURL}8/html-single/configuring_and_managing_identity_management/index#con_host-entry-LDAP_managing-hosts-ui[Host entry in IdM LDAP] in _Configuring and managing Identity Management_.
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc:32]:
```suggestion
For more information on host configuration properties, see {RHELDocsBaseURL}8/html-single/configuring_and_managing_identity_management/index#con_host-entry-LDAP_managing-hosts-ui[Host entry in IdM LDAP] in _{RHEL}{nbsp}8 Configuring and managing Identity Management_.
```

File: guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc [hunk 7]
```diff
@@ -39,7 +39,7 @@ endif::[]
 ----
 +
 ifdef::satellite[]
-For more information on managing services, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/accessing_identity_management_services/index[Accessing Identity Management services] in _{RHEL} guide_.
+For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[Accessing Identity Management services] in _{RHEL} guide_.
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc:42]:
```suggestion
For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[_{RHEL}{nbsp}9 Accessing Identity Management services_].
```

File: guides/common/modules/proc_verifying-dns-resolution.adoc [hunk 8]
```diff
@@ -43,7 +43,7 @@ rtt min/avg/max/mdev = 0.019/0.019/0.019/0.000 ms
 ----
 
 ifdef::satellite[]
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_networking/assembly_changing-a-hostname_configuring-and-managing-networking#proc_changing-a-hostname-using-hostnamectl_assembly_changing-a-hostname[Changing a hostname using hostnamectl] in the _{RHEL} Configuring and managing networking_ guide.
+For more information, see {RHELDocsBaseURL}9/html-single/configuring_and_managing_networking/index#proc_changing-a-hostname-using-hostnamectl_assembly_changing-a-hostname[Changing a hostname using hostnamectl] in the _{RHEL} Configuring and managing networking_ guide.
```
Comment (Lennonka) [inline guides/common/modules/proc_verifying-dns-resolution.adoc:46]:
```suggestion
For more information, see {RHELDocsBaseURL}9/html-single/configuring_and_managing_networking/index#proc_changing-a-hostname-using-hostnamectl_assembly_changing-a-hostname[Changing a hostname using hostnamectl] in _{RHEL} Configuring and managing networking_.
```

**Review by Lennonka — CHANGES_REQUESTED** [review]
Further suggestions.

File: guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc [hunk 9]
```diff
@@ -39,7 +39,7 @@ endif::[]
 ----
 +
 ifdef::satellite[]
-For more information on managing services, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/accessing_identity_management_services/index[Accessing Identity Management services] in _{RHEL} guide_.
+For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[Accessing Identity Management services] in the _{RHEL} guide_.
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc:42]:
```suggestion
For more information on managing services, see {RHELDocsBaseURL}9/html/accessing_identity_management_services/index[_{RHEL}{nbsp}9 Accessing Identity Management services_].
```

File: guides/common/modules/proc_configuring-host-based-authentication-control.adoc [hunk 10]
```diff
@@ -5,7 +5,7 @@ HBAC rules define which machine within the domain a {FreeIPA} user is allowed to
 You can configure HBAC on the {FreeIPA} server to prevent selected users from accessing {ProjectServer}.
 With this approach, you can prevent {Project} from creating database entries for users that are not allowed to log in.
 ifndef::orcharhino[]
-For more information on HBAC, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_idm_users_groups_hosts_and_access_control_rules/index[Managing IdM users, groups, hosts, and access control rules] in _{RHEL} 9 guide_ or https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_idm_users_groups_hosts_and_access_control_rules/index[Managing IdM users, groups, hosts, and access control rules] in _{RHEL} 8 guide_.
+For more information on HBAC, see {RHELDocsBaseURL}9/html/managing_idm_users_groups_hosts_and_access_control_rules/index[Managing IdM users, groups, hosts, and access control rules] in the _{RHEL}{nbsp}9 guide_ or {RHELDocsBaseURL}8/html/managing_idm_users_groups_hosts_and_access_control_rules/index[Managing IdM users, groups, hosts, and access control rules] in the _{RHEL}{nbsp}8 guide_.
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-host-based-authentication-control.adoc:8]:
```suggestion
For more information on HBAC, see {RHELDocsBaseURL}9/html/managing_idm_users_groups_hosts_and_access_control_rules/index[_{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules_] or {RHELDocsBaseURL}8/html/managing_idm_users_groups_hosts_and_access_control_rules/index[_{RHEL}{nbsp}8 Managing IdM users, groups, hosts, and access control rules_].
```

File: guides/common/modules/proc_configuring-repositories-proxy.adoc [hunk 11]
```diff
@@ -58,18 +56,15 @@ include::snip_step-verify-enabled-repolist.adoc[]
...
-[NOTE]
-====
-If there is any warning about conflicts with Ruby or PostgreSQL while enabling `{dnf-module}` module, see xref:troubleshooting-dnf-modules_{context}[Troubleshooting DNF modules].
-For more information about modules and lifecycle streams on {RHEL} 8, see https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle[{RHEL} Application Streams Lifecycle].
-====
+include::snip_step-verify-enabled-repolist.adoc[]
+
+.Additional Resources
+* If there is any warning about conflicts with Ruby or PostgreSQL while enabling `{dnf-module}` module, see xref:troubleshooting-dnf-modules_{context}[Troubleshooting DNF modules].
+* For more information about modules and lifecycle streams on {RHEL}{nbsp}8, see https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle[{RHEL} Application Streams Lifecycle].
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-repositories-proxy.adoc:70]:
```suggestion
* For more information about modules and lifecycle streams on {RHEL}{nbsp}8, see https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle[{RHEL}{nbsp}8 Application Streams Lifecycle].
```

File: guides/common/modules/proc_verifying-dns-resolution.adoc [hunk 12]
```diff
@@ -43,7 +43,7 @@ rtt min/avg/max/mdev = 0.019/0.019/0.019/0.000 ms
 ----
 
 ifdef::satellite[]
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_networking/assembly_changing-a-hostname_configuring-and-managing-networking#proc_changing-a-hostname-using-hostnamectl_assembly_changing-a-hostname[Changing a hostname using hostnamectl] in the _{RHEL} Configuring and managing networking_ guide.
+For more information, see {RHELDocsBaseURL}9/html-single/configuring_and_managing_networking/index#proc_changing-a-hostname-using-hostnamectl_assembly_changing-a-hostname[Changing a hostname using hostnamectl] in _{RHEL} Configuring and managing networking_.
```
Comment (Lennonka) [inline guides/common/modules/proc_verifying-dns-resolution.adoc:46]:
```suggestion
For more information, see {RHELDocsBaseURL}9/html-single/configuring_and_managing_networking/index#proc_changing-a-hostname-using-hostnamectl_assembly_changing-a-hostname[Changing a hostname using hostnamectl] in _{RHEL}{nbsp}9 Configuring and managing networking_.
```

File: guides/common/modules/ref_system-requirements.adoc [hunk 13]
```diff
@@ -107,14 +107,14 @@ ifndef::satellite[]
...
+For more information, see {RHELDocsBaseURL}9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode] in _{RHEL}{nbsp}9 Security hardening_ or {RHELDocsBaseURL}8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening[Installing the system in FIPS mode] in _{RHEL}{nbsp}8 Security hardening_.
 
 [NOTE]
 ====
 {Project} supports DEFAULT and FIPS crypto-policies.
 The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
 The FUTURE policy is a stricter forward-looking security level intended for testing a possible future policy.
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening[Using system-wide cryptographic policies] in the {RHEL} guide.
+For more information, see {RHELDocsBaseURL}9/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening[Using system-wide cryptographic policies] in _{RHEL} Security hardening_.
```
Comment (Lennonka) [inline guides/common/modules/ref_system-requirements.adoc:117]:
```suggestion
For more information, see {RHELDocsBaseURL}9/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening[Using system-wide cryptographic policies] in _{RHEL}{nbsp}9 Security hardening_.
```

File: guides/common/modules/snip_step-verify-enabled-repolist.adoc
Comment (Lennonka) [inline guides/common/modules/snip_step-verify-enabled-repolist.adoc:1]: @mjivraja I'm so sorry, I've missed that you've modified the content and added the Verification heading! In such case, you were right to change the name of the snippet to `snip_verification-*`.

File: guides/common/modules/ref_system-requirements.adoc [hunk 14]
```diff
@@ -35,12 +35,12 @@ endif::[]
 
 {Project} only supports `UTF-8` encoding.
 If your territory is USA and your language is English, set `en_US.utf-8` as the system-wide locale settings.
-For more information about configuring system locale in {EL}, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring the system locale].
+For more information about configuring system locale in {EL}, see {RHELDocsBaseURL}9/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring the system locale] in _Configuring basic system settings_.
```
Comment (Lennonka) [inline guides/common/modules/ref_system-requirements.adoc:38]:
```suggestion
For more information about configuring system locale in {EL}, see {RHELDocsBaseURL}9/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring the system locale] in _{RHEL}{nbsp}9 Configuring basic system settings_.
```

**Review by Lennonka — CHANGES_REQUESTED** [review]
A couple of details and we'll be good to go!

File: guides/common/modules/snip_step-verify-enabled-repolist.adoc
Comment (mjivraja) [reply guides/common/modules/snip_step-verify-enabled-repolist.adoc:1]: No worries Lena, I have reverted :+1:

**Review by Lennonka — APPROVED** [review]
Awesome, thank you!

**Comment by Lennonka** [comment]
There are cherry-pick conflicts on 3.10. I personally don't mind if it doesn't get cherry-picked.

**Comment by mjivraja** [comment]
> There are cherry-pick conflicts on 3.10. I personally don't mind if it doesn't get cherry-picked.

I think we may need to cherry-pick this until `3.3` because domain change will affect each version.

**Comment by Lennonka** [comment]
> > There are cherry-pick conflicts on 3.10. I personally don't mind if it doesn't get cherry-picked.
> 
> I think we may need to cherry-pick this until `3.3` because domain change will affect each version.

We don't, because there will be redirects.

**Comment by mjivraja** [comment]
> We don't, because there will be redirects.

Gotcha, thanks :+1:

**Pull request merged**